### ISSUE TITLE
Holding shift/control and using scroll wheel should zoom in image view

### ISSUE THREAD

**Issue by meyer9** [nion-software/nionswift#7]

**Comment by cmeyer** [comment]
AL 2020-03-11:

The way I think Swift should let you navigate around images with a three button mouse is:
That the middle button (which doubles as a scroll wheel) should vary the zoom. Pressing and holding the middle button should let you navigate around the image without changing the zoom.

The way that Blender lets you navigate around a 3D object is similar. (Technically the middle button lets you rotate. You have to hold shift to translate. Control gets you back to zoom.) Obviously it's a bit different here as we are interested in 2D images. Mostly.

**Comment by Brow71189** [comment]
+1 on this. This is also how other (2D) software that I use works.

Am Di., 20. Okt. 2020 um 18:26 Uhr schrieb Chris Meyer <
notifications@github.com>:

**Comment by ndellby** [comment]
This is ignoring that we usually use trackballs without scroll wheels, 
because they work miles better for adjusting knobs.

The track balls have little weird mini buttons, no third button and no 
little wheel.

Sometimes Gwyn adds a mouse as well as the trackball, to operate you 
have to hang the mouse in the air so it doesn't nudge controls. So dual 
three button mouse and track ball is also not practical.

Niklas

**Comment by cmeyer** [comment]
I think the trackball is just treated as a mouse. There is a separate path for detecting changes to the scroll wheels on mouses (some mice have multiple scroll wheels btw) and for detecting 2nd and 3rd button mouse presses.

**Comment by ndellby** [comment]
Features that don't work on a trackball are likely to be worse than useless, you get used to them off-line, and then when you're on the microscope they don't work.

**Comment by Brow71189** [comment]
I disagree with that. This feature would not cause any issues with a trackball because you physically cannot do the action it takes to trigger it. So there is no chance of confusion or weird things happening when you're attempting to zoom with the mouse wheel because you simply don't have one available.

Another option, which could be generally useful, would be to get a trackball with scrollwheel. For example one of the following:

https://www.amazon.com/Kensington-Expert-Wireless-Trackball-K72359WW/dp/B01936N73I/ref=sr_1_6?dchild=1&keywords=trackball&qid=1603810109&sr=8-6

https://www.amazon.com/ELECOM-M-HT1DRBK-Wireless-Trackball-Mouse/dp/B0735584RM/ref=sr_1_10?dchild=1&keywords=trackball&qid=1603810023&sr=8-10

https://www.amazon.com/Kensington-Orbit-Trackball-Scroll-K72337US/dp/B002OOWB3O/ref=sr_1_8?dchild=1&keywords=trackball&qid=1603810023&sr=8-8

**Comment by Brow71189** [comment]
soo also #50 which is closely related

**Comment by jamesrussell216** [comment]
c.f. https://github.com/nion-software/nionswift/issues/1625
